install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -38,8 +38,6 @@ spec:
           args:
           - proxy
           - router
-          - -v
```

Comment:
Instead of removing this, I prefer that you use `log_output_level` instead.

Reply:
sgtm